tests/saitests/py3/sai_qos_tests.py

```diff
@@ -1711,6 +1731,9 @@ def runTest(self):
         if 'pkts_num_egr_mem' in list(self.test_params.keys()):
             pkts_num_egr_mem = int(self.test_params['pkts_num_egr_mem'])
 
+        if 'is_multi_asic' in list(self.test_params.keys()):
+            is_multi_asic = bool(self.test_params['is_multi_asic'])
```

Comment:
Variable `is_multi_asic` appears to be undefined in negative case.

Could this use dict default values to avoid the 'if' stmt?
```py
is_multi_asic = bool(self.test_params.get('is_multi_asic', False))
```

Reply:
Updated.